<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistique</class>
 <widget class="QWidget" name="Statistique">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>580</y>
     <width>176</width>
     <height>31</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>661</width>
     <height>574</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="grid">
    <item row="15" column="0">
     <widget class="QLabel" name="lab_rev5">
      <property name="text">
       <string>&gt; 40001 €</string>
      </property>
      <property name="indent">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="line_rev3"/>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="line_rev5"/>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="line_rev1"/>
    </item>
    <item row="14" column="1">
     <widget class="QLineEdit" name="line_rev4"/>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="conso_marque">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Revenu</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="lab_rev1">
      <property name="text">
       <string>&lt; 10000 €</string>
      </property>
      <property name="indent">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="line_rev2"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_enfant">
      <property name="text">
       <string>nombre d'enfant</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_stat">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Statistiques</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <layout class="QGridLayout" name="grid_marque">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupe_marque">
        <property name="title">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Marque</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_prop">
      <property name="text">
       <string>proportion de yaourt</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="lab_rev3">
      <property name="text">
       <string>20001 à 30000 €</string>
      </property>
      <property name="indent">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="lab_rev2">
      <property name="text">
       <string>10000 à 20000 €</string>
      </property>
      <property name="indent">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="nb_enfant">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="conso_type">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="tranch_age">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="lab_rev4">
      <property name="text">
       <string>30001 à 40000 €</string>
      </property>
      <property name="indent">
       <number>3</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_conso">
      <property name="text">
       <string>consommation moyenne</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_tranche">
      <property name="text">
       <string>tranche d'age</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <layout class="QGridLayout" name="grid_type">
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupe_type">
        <property name="title">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
        <property name="indent">
         <number>3</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="16" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Pourcentage Femme</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QLineEdit" name="line_femme"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
